recipe/meta.yaml

```
@@ -10,22 +10,25 @@ source:
   sha256: e292605effc7da39b7a8734c719afb12ec4b5362add3528d8afad3aa3aa9057c
 
 build:
-  number: 2
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --verbose"
-  skip: True  # [win and vc<14]
+  number: 3
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
-    - pip >=18.1
-    - grpcio >=1.16.0
-    - protobuf >=3.6.1
+    - pip
+    - setuptools
+    - wheel
   run:
     - python
-    - {{ pin_compatible('grpcio') }}
-    - {{ pin_compatible('protobuf') }}
+    - grpcio >=1.16.0
```

```suggestion
    - grpcio
```
I'm not seeing where this version comes from and it seems the oldest version we have with all five Python versions supported is `1.48.2`

---

I'm also not seeing an explicit dependency on `protobuf`

---

Nvm about `protobuf`. I see the change on the main branch